op-guide/tidb-v2.1-upgrade-guide.md

```diff
@@ -5,7 +5,7 @@ category: deployment
 
 # TiDB 2.1 升级操作指南
 
-本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 rc 版本升级到 TiDB 2.1 GA 版本。2.1 版本 TiDB-Binlog 不兼容 Kafka 版本，如果当前集群已经使用 Kafka 版本的 TiDB-Binlog，升级需要参考 [Binlog 升级方法](../tools/tidb-binlog-cluster.md#版本升级方法)。
+本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 rc 版本升级到 TiDB 2.1 GA 版本。2.1 版本 TiDB 不兼容 Kafka 版本的 TiDB-Binlog，如果当前集群已经使用 Kafka 版本的 TiDB-Binlog，升级需要参考 [Binlog 升级方法](../tools/tidb-binlog-cluster.md#版本升级方法)。
```

Comment:
- What if the user wants to upgrade to TiDB 2.1.1? 
- Perhaps a link to the guide of the Kafka version of TiDB-Binlog is needed? Otherwise, the user might have no idea about what is the Kafka version.

Reply:
本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 rc 版本升级到 TiDB 2.1 GA _**及以上**_版本
这样可以吗